package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scripts": {
+    "test": "jest src/**.test.js --watch",
```

Comment:
can you do a `npm run watch` and leave `test`  for a basic run?

Reply:
yeah, sure.

Reply:
done